src/main/java/hudson/plugins/sauce_ondemand/SauceEnvironmentUtil.java

```diff
@@ -213,7 +210,7 @@ public static SauceOnDemandBuildWrapper getBuildWrapper(AbstractProject<?, ?> pr
      * @return String representing the Jenkins build
      */
     @Nonnull
-    public static String getBuildName(AbstractBuild<?, ?> build) {
+    public static String getBuildName(Run<?, ?> build) {
```

Comment:
Yeah that is interesting, I have always seen it for maps `Map<K,V>`, you are right ! :).